sandbox/auth/session/idclaims.go

```diff
@@ -0,0 +1,63 @@
+package session
+
+import "github.com/go-jose/go-jose/v3/jwt"
+
+// Standard claims:
+// oidc: https://openid.net/specs/openid-connect-basic-1_0.html#IDToken
+// jwt: https://datatracker.ietf.org/doc/html/rfc7519#section-4
+// oidc user: https://openid.net/specs/openid-connect-basic-1_0.html#StandardClaims
+
+type IDClaims struct {
+	ID string `json:"jti,omitempty"`
+
+	Issuer    string           `json:"iss,omitempty"`
+	Subject   string           `json:"sub,omitempty"`
+	Audience  jwt.Audience     `json:"aud,omitempty"`
+	Expiry    *jwt.NumericDate `json:"exp,omitempty"`
+	NotBefore *jwt.NumericDate `json:"nbf,omitempty"`
+	IssuedAt  *jwt.NumericDate `json:"iat,omitempty"`
+
+	Nonce           string `json:"nonce,omitempty"`
+	AccessTokenHash string `json:"at_hash,omitempty"`
+
+	// Email scope:
+
+	Email         string `json:"email,omitempty"`
+	EmailVerified bool   `json:"email_verified,omitempty"`
+
+	// Profile scope:
+
+	Name       string           `json:"name,omitempty"`
+	GivenName  string           `json:"given_name,omitempty"`
+	FamilyName string           `json:"family_name,omitempty"`
```

Comment:
not first name, last name?

Reply:
The names come from the standard: https://openid.net/specs/openid-connect-basic-1_0.html#StandardClaims

Reply:
Should I keep only name for now?

Reply:
I'm fine keeping them if it is part of the standard 🤷‍♀️